src/Clap-Core.package/ClapAlternative.class/README.md

```diff
@@ -0,0 +1,19 @@
+I am a class which represents all the possible ways to execute a command.
+
+For example,  imagine you have two ways to execute the command eval with the help flag, the first one is "eval -h <exp>" and the second one is "eval --help <exp>".You create a ClapCommand for the first: 
+	c1 := (ClapCommand withName: 'eval')
+		addFlag: ClapFlag withName: 'h';
+		...
+		
+And the second:  
+	c2 := (ClapCommand withName: 'eval')
+		addFlag: ClapFlag withName: 'help';
+		...
+
+And you create your ClapAlternative like this:
+	(ClapAlternative new) add: c1;add: c2.	
+
+		
+This class stocks these commands in an OrderedCollection called alternatives, so the first ClapCommand you add to the ClapAlternative will be evaluated first and so forth.
```

Comment:
Not evaluated! This is at *parse* time! Clap will try to recognize the alternatives in order and the first one to succeed will be added to the result